pom.xml

```diff
@@ -66,6 +67,11 @@
 			<artifactId>spring-dotenv</artifactId>
 			<version>${spring-dotenv.version}</version>
 		</dependency>
+		<dependency>
+			<groupId>org.mapstruct</groupId>
+			<artifactId>mapstruct</artifactId>
+			<version>1.5.5.Final</version>
```

Comment:
why define property for mapstruct version, and then use different once in dependencies?